### ISSUE TITLE
style.conf settings ignored

### ISSUE THREAD

**Issue by Konfuz** [zim-desktop-wiki/zim-desktop-wiki#508]
On a manjaro Linux I try to change my color style so that It works with the default dark Theme of my distribution (which it unfortunately does not out of the box due to the dark link color)

When I edit the `~/.config/zim/style.conf` nothing seems to have an effect on my existing Notebook (which is located in a non-default path) even after restarting zim and/or pressing `ctrl-R` I even tried temporarily deleting the conf.style along with the master version in `/usr/share/zim/`which did not bother zim at all. 

However: If I generate a new Notebook it will apply my style changes to the new Notebook. If I **then** press `strg+R` on my old Notebook it will apply the changes there as well. There seems to be some weird caching going on and It took me 2 days to figure out how to change the Colors.

**Comment by Osndok** [comment]
Perhaps you have the wrong path? On my machine it is ```~/.config/zim/style.conf```

**Comment by Konfuz** [comment]
> Perhaps you have the wrong path? On my machine it is ~/.config/zim/style.conf
Sorry, that was a typo in my original comment I edited it out for clarity. I am indeed talking about ~/.config/zim/style.conf

**Comment by radekh** [comment]
I installed Zim from deb package from Zim web.
This behaviour of style.conf is more annoying than Konfuz described.
Not only it ignores the style.conf I copied in place from my archive/backup.
It also rewrites this file by some defaults whenever I touched preferences menu.

**Comment by radekh** [comment]
It seems that making the style.conf read only helps a much.

**Comment by jaap-karssenberg** [comment]
Tested with latest master and seems to work correctly. Do note you need to restart zim for changes in style.conf to take effect (and that includes closing the trayicon).

Please re-open if seen in zim 0.70 or newer

**Comment by JohnRobson** [comment]
I have the same problem, style.conf is ignored.

**Comment by theflyingswami** [comment]
I am having this issue in 2023. Flatpak install on PopOs 22.04. 

I copied style.conf from the Zim flatpak directory I found in /.local/share/flatpak/app/[verylongpath]... to .config/zim/style.conf and changed [Tag link] from 'blue' to 'light blue', then restart Zim, but nothing changes.

I've also tried substituting 'light blue' with #85e7f3 and altering the file located in the .local/share/flatpak/... path, to no avail. Zim ignores everything.

The Zim manual does not address Flatpak installs, so maybe I'm doing something wrong, idk.
